tests/monster_test_generated.ts

```diff
@@ -87,10 +87,15 @@ static endInParentNamespace(builder:flatbuffers.Builder):flatbuffers.Offset {
   return offset;
 };
 
+/**
```

Comment:
Those parameter comments in TS are quite useless and only add noise.

Reply:
Remove the comments.  The generated TypeScript is now unchanged.